src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Colors[] colors = Colors.values();
+        Random random = new Random();
```

Comment:
Random object creates every time you call getRandomColor. Think of the approach, where you create an object of Random only once for this class